apps/frontend/src/components/global/upload_tabs/LoadFileList.vue

```diff
@@ -1,16 +1,47 @@
 <template>
   <div>
-    <v-container>
+    <v-container class="ml-0 pl-0 mr-0 pr-0 pb-0 mb-0">
```

Comment:
Is the intent here to remove all margin and padding except the top? If so this could just be `my-0 py-0 pb-0 mb-0` or if this intended to be all directions `ma-0 pa-0`